index.html

```diff
@@ -48,133 +43,151 @@ <h2>
         <li>Versicherungen</li>
         <li>Betriebswirtschaftslehre</li>
     </ul>
-</div>    
+</div>
+<div class="sections about white">
+    <h5>Über mich</h5>
+    <p class="profiltxt">
+        Geboren bin ich 1988 in Frankfurt am Main, 
+        aufgewachsen in Eschborn und Hattersheim im Main-Taunus-Kreis. 
+        Nach meinem Realschulabschluss ging ich erstmal als Zeitsoldat 
+        zu den Herresfliegern der Bundeswehr.</p>
+    <p class="profiltxt"> Aufgrund einer Erkrankung 
+        musste ich leider vorzeitig aus dem Dienst aus-scheiden. 
+        Anschließend schloss ich eine Ausbildung zum Kaufmann für Versicherungen 
+        und Finazen ab. Darauf folgten 10 Jahre in der Versicherungs-branche in Positionen, 
+        als Produkt-manager oder Fachspezialist für gewerbliche Versicherungen.</p> 
+    <p class="profiltxt">Vor 4 Jahren entschied ich mich zusätz-lich noch BWL zu studieren. Ich schloss das Studium Ende 
+        2020 erfolgreich mit einem Notendurchschnitt von 1,9 ab. Nebenbei betätige ich 
+        mich  als selbstständiger Versicherungsmakler.
+    </p>
+</div>
+<div class="sidebar display-in">
+    <div class="sidebar-table">
+        <a class="sidebar-link" href="#contact"><button>Kontakt</button></a>
+        <div class="sidespace"></div>
+        <a class="sidebar-link" href="#biography"><button>Biografie</button></a>   
+ </div>      
 </section>
 
-<section id="contact">
-<div>
-   <h3>Hier sind meine Kontaktdaten</h3>  
-    <table class="contact">
-        <tr>
-            <td>Mail:</td>
-            <td>xtybevip@googlemail.com</td>
-        </tr>
-        <tr>
-            <td>Handy-Nr.:</td>
-            <td>0152/53453231</td>
-        </tr>
-    </table>
-    <form action="">
-        <div>
-            <label for="name">Nachname:</label>
-            <input type="text" name="" id="name">
-        </div>
-        <div>
-            <label for="vorname">Vorname:</label>
-            <input type="text" name="" id="vorname">
+<section>
+    <a href="https://github.com/stefankorder" target="_blank">
+        <div class="profile white">
+            <img src="images/github.webp" alt="" width="250">
```

Comment:
Reconsider this structure. Do you really need the `div` around the img? This is a bit of unusual to have this "extra" div between anchor tag and img.